docs/notebooks/bsync_examples/Small-Office-Level-2.md

```diff
@@ -0,0 +1,2179 @@
+```xml
+<BuildingSync>
+    <Facilities>
+        <Facility ID="F-1">
+            ...children stuff here...
+        </Facility>
+        <Facility ID="F-2">
+            ...children stuff here...
+        </Facility>
+    </Facilities>
+</BuildingSync>
+```
+- While the above can be done, it is __most common__ to find a single Facility, Site, and Building in a BuildingSync document. In general:
+    - Facility refers to a grouping of Sites, but a Facility is still geographically proximate. Its boundary might include not building specific things (streets, sidewalks, etc.)
+    - A Site is everything in and around a group of Buildings. This might be two Buildings that are connected via a tunnel, etc.
+    - ** Most common is just to have a single Building
+
+### Primary BuildingSync Structural Elements
+We start by creating the primary structure of a BuildingSync document. We introduced `Facilities`, `Sites`, and `Buildings`, but there are a few additional:
+- `Sections` are subportions of a `Building`. Think of a mixed-use commercial building with retail and office space, each of these would become a section. For those familiar with the ASHRAE 211 Normative Spreadsheet, a `Section` is analagous to a column in the `All - Space Functions` sheet.
+- `Systems` are defined under a `Facility` and capture the energy consuming / producing assets. Each can be linked back to an individual `Facility`, `Site`, `Building`, or `Section`.
+- `Reports` are defined under a `Facility` and each `Report` would contain information about reporting type requirements. Most importantly, a `Report` contains a set of `Scenario` elements, where each `Scenario` would convey building performance for one of: current / historical data, target performance, benchmark performance, baseline modeled performance, or expected performance with measure(s) implemented.
+
+
+```python
+# just take this for what it is at the moment. the bsync package makes
+# it easier to work with xml content. All of this is mainly syntactic
+# sugar around the lxml package, with some type checking, attribute 
+# assignment, etc.
+root = bsync.BuildingSync()
+facs = bsync.Facilities()
+f1 = bsync.Facilities.Facility(ID="Facility-1")
+sites = bsync.Sites()
+s1 = bsync.Sites.Site(ID="Site-1")
+bldgs = bsync.Buildings()
+b1 = bsync.Buildings.Building(ID="Building-Small-Office-Prototype")
+sections = bsync.Sections()
+systems = bsync.Systems()
+contacts = bsync.Contacts()
+reports = bsync.Reports()
+measures = bsync.Measures()
+schedules = bsync.Schedules()
+utilities = bsync.Utilities()
+
+# We link the entire report to the building
+r1 = bsync.Report(
+    bsync.LinkedPremisesOrSystem(
+        bsync.LinkedPremisesOrSystem.Building(
+            bsync.LinkedBuildingID(IDref=b1["ID"])
+        )
+    ),
+    ID='Report-L2-Audit')
```

Comment:
There is `ASHRAEAuditLevel` that can be set here too. The syntax would be something like (confirm that it works). Also, if it does work, then add to Level 1 as well, please. (Element is [here](https://buildingsync.net/dictionary/2.4.0/#59990)).

```
r1 = bsync.Report(
     bsync.ASHRAEAuditLevel("Level 2: Energy Survey and Analysis")
     bsync.LinkedPremisesOrSystem(
         bsync.LinkedPremisesOrSystem.Building(
             bsync.LinkedBuildingID(IDref=b1["ID"])
         )
     ),
     ID='Report-L2-Audit')
```